### ISSUE TITLE
Window size & window shadow issue

### ISSUE THREAD

**Issue by cmcmenamin** [superhighfives/pika#238]
### Window shadow can cause inaccurate colour readings

I use Pika frequently to assess formal accessibility compliance. I notice sometimes when I'm using the colour picker, if I'm not careful, the shadow around the Pika window can actually influence colour picking results — so I have to be very careful. I have noticed I can enable the "Hide window while picking" option in the Settings and that kind of fixes it, but perhaps Pika should also default to having no window shadow, or make the shadow optional? The shadow extends out quite far...

### Minimum window size got..bigger

I noticed the latest version of Pika has a larger minimum window size i.e. the vertical height is greater.

This means it takes up more space on my display than is necessary — I loved Pika for how unobtrusive and compact it is; I'm wondering if Pika can be updated so it can become far smaller and less obtrusive.

**Comment by superhighfives** [comment]
Hey @cmcmenamin. Thanks for the thoughtful feedback. To be honest, I've been a little worried about Pika getting too chonk for its utility, and the reason I did that was because I wanted to be able to reasonably fit having every view turned on at once at the smallest size. But I also think I can just make stuff auto hide at smaller sizes if it gets too cluttered.

Which is a long way of saying, on it for 1.9.0. Cheers for reaching out.

**Comment by cmcmenamin** [comment]
All good 😎 Love this app so much by the way 😀

**Comment by superhighfives** [comment]
Sweet, this is now available in the `1.9.0` beta, which you can check out here: https://github.com/superhighfives/pika/releases/tag/1.9.0-beta2. You can also download it from within Pika's settings by enabling beta releases and checking for updates. Let me know what you think, @cmcmenamin.

**Comment by cmcmenamin** [comment]
Ooooh nice work 😎

- The setting to remove window shadow works - this is purely aesthetic, but I wonder if the window needs a subtle border in this mode so it doesn't blend in with what is behind the window?
- Hide shadow on colour picking option does not seem to work for me - shadow remains visible while in colour picking mode and the shadow influences picked colour values

I like that I can make the window more compact but not how the colour contrast ratio section vanishes at small window sizes, I reckon there's more than enough vertical space for it to remain expanded. I use Pika for WCAG audits so... the contrast bit is the most important to me 😀

I did some more clicking around and I kinda see the space issue you're trying to solve - the new text preview feature can take up space. Perhaps the contrast section should only collapse if the text preview is enabled... so stuff is only hidden if there actually is a space constraint or overlap risk. I don't know 🤷‍♂️ 😂

I wonder, as a product decision, how window shadow should be handled. I suspect one of the main reasons people use Pika is for WCAG  compliance checking, where absolute precision measurements are a 'must', or else you report false positive failures 🤔 I wonder if people are accidentally getting incorrect measurements without realising - this is what was happening for me when I realised the shadow was interfering with measurements. Perhaps it's good enough to simply have an opt-in hide shadow mode, I'm unsure.

**Comment by superhighfives** [comment]
Got a border in there now, and it gets reeeeeeeal tiny now (while still keeping the left side of the footer).

<img width="477" height="345" alt="Image" src="https://github.com/user-attachments/assets/5165a515-1e19-4752-95f4-33c1a8d75fba" />

<img width="477" height="345" alt="Image" src="https://github.com/user-attachments/assets/c4fb333e-4a6e-412b-8a12-7aa8b24947d4" />

Cutting a new release now, `1.9.0-beta3`.

**Comment by cmcmenamin** [comment]
beautiful 😌
